.github/workflows/benchmark.yml

```
@@ -0,0 +1,20 @@
+name: Benchmarks
+on:
+  pull_request_target:
```

Pretty sure this workflow doesn't work because of this line. Shouldn't it be:

```suggestion
  pull_request:
```

---

It is my understanding that `pull_request` runs in the context of this repo and `pull_request_target` runs in the context of the fork. This option was mentioned in the discussion of permissions on the [comment action](https://github.com/marketplace/actions/comment-pull-request#permissions). I am less familiar with github actions, in general, so advice is definitely appreciated.

---

Ooh, I didn't know this was a thing. Then yeah, seems reasonable that this will only work after merging. Same as the `workflow_run` trigger.